cockatrice/src/tab_deck_editor.cpp

```diff
@@ -1248,7 +1249,7 @@ void TabDeckEditor::showSearchSyntaxHelp()
                .replace(QRegularExpression("^(##)(.*)", opts), "<h2>\\2</h2>")
                .replace(QRegularExpression("^(#)(.*)", opts), "<h1>\\2</h1>")
                .replace(QRegularExpression("^------*", opts), "<hr />")
-               .replace(QRegularExpression("\\[([^\[]+)\\]\\(([^\\)]+)\\)", opts), "<a href=\'\\2\'>\\1</a>");
+               .replace(QRegularExpression(R"(\[([^[]+)\]\(([^\)]+)\))", opts), R"(<a href='\2'>\1</a>)");
```

Comment:
I don't think this is a readability improvement when all the others use the escaped backslashes

Reply:
This wasn't readable to begin with, but the linter recommended I change it. It no longer has the incorrect escape sequence, which was lost in all the backslashes to begin with.

Reply:
you mean the escaped [?

Reply:
Yeah, which wasn't necessary